### ISSUE TITLE
모델 성능 향상

### ISSUE THREAD

**Issue by kimtaewan22** [kimtaewan22/NLP_Model#2]

**Comment by kimtaewan22** [comment]
loss function을 다른 것으로 변경하여 진행

**Comment by kimtaewan22** [comment]
뷸균형 데이터를 해결할 수 있는 방법
- 샘플링 방식 적용
- loss function - weighted cross entropy, focal loss

**Comment by kimtaewan22** [comment]
weighted cross entropy, focal loss를 적용했을 때 큰 성능 향상을 보이지 않음

**Comment by kimtaewan22** [comment]
긍정 텍스트에 대한 데이터를 추가하여 진행
